app/src/main/java/com/omelan/cofi/MainActivity.kt

```diff
@@ -121,6 +123,9 @@ class MainActivity : MonetCompatActivity() {
         RecipeDetails(
             recipeId = recipeId,
             onRecipeEnd = { recipe ->
+                if (isInstantApp()) {
```

Comment:
With change to the `isInstantApp` signature you would need to add
```suggestion
                if (isInstantApp(this)) {
```
to pass the context of an Activity there